types/react-native-actionsheet/index.d.ts

```diff
@@ -7,16 +7,30 @@
 import * as React from 'react';
 
 export interface ActionSheetProps {
-    options: React.ReactNode[];
+    options: string[];
     onPress: (index: number) => void;
     title?: string;
     message?: string;
     tintColor?: string;
     cancelButtonIndex?: number;
     destructiveButtonIndex?: number;
+}
+
+export interface ActionSheetCustomProps {
```

Comment:
how about `buttonUnderlayColor`?

Reply:
Good catch! It's undocumented, so I did not even notice it.

Reply:
I have added it now.

Reply:
@ian-rudge can you please re-review?